pkg/sourcetransformer/examples/event_time_filter/Makefile

```diff
@@ -1,18 +1,19 @@
 TAG ?= stable
 PUSH ?= false
+IMAGE_REGISTRY = quay.io/numaio/numaflow-go/mapt-event-time-filter:${TAG}
```

Comment:
Just curious, if we update the branch to bring in Abdullah's change, should these Makefile changes NOT show up in the diff?

Reply:
I did pull in his changes into this branch. I'm guessing that should be taken care of while merging in